.drone.yml

```diff
@@ -0,0 +1,116 @@
+pipeline:
+
+  build-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    commands:
+      - docker build -t hocs-management-ui .
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
+      event: push
+
+  install-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    secrets:
+      - docker_password
+    commands:
+      - docker login -u="ukhomeofficedigital+hocs" -p=$${DOCKER_PASSWORD} quay.io
+      - docker tag hocs-management-ui quay.io/ukhomeofficedigital/hocs-management-ui:build-$${DRONE_BUILD_NUMBER}
+      - docker tag hocs-management-ui quay.io/ukhomeofficedigital/hocs-management-ui:latest
+      - docker push quay.io/ukhomeofficedigital/hocs-management-ui:build-$${DRONE_BUILD_NUMBER}
+      - docker push quay.io/ukhomeofficedigital/hocs-management-ui:latest
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
+      event: push
+
+  docker-semver-tag:
+    image: quay.io/ukhomeofficedigital/hocs-version-bot:build-25
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+      - DOCKER_API_VERSION=1.37
+    secrets:
+      - github_password
+      - docker_password
+      - git_password
+    commands:
+      - /app/hocs-deploy --version=$${SEMVER} --serviceGitToken=$${GIT_PASSWORD} --service=hocs-management-ui --gitToken=$${GITHUB_PASSWORD} --gitRepo="https://gitlab.digital.homeoffice.gov.uk/hocs/hocs-versions.git" --environment=qa --dockerRepository=quay.io/ukhomeofficedigital --sourceBuild=$${IMAGE_VERSION} --registryUser=ukhomeofficedigital+hocs --registryPassword=$${DOCKER_PASSWORD}
```

Comment:
Might be more sensible to put these as environment variables?

Reply:
Removed the sem-ver step when deploying to qa